packages/paste-website/src/components/site-wrapper/site-header/DarkModeBadge.tsx

```diff
@@ -1,26 +1,31 @@
-import * as React from 'react';
-import {Box} from '@twilio-paste/box';
-import {Stack} from '@twilio-paste/stack';
-import {Modal, ModalBody, ModalHeader, ModalHeading} from '@twilio-paste/modal';
-import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
-import {DarkModeFAQModal} from './DarkModeFAQModal';
-import {DarkModeBadgeButton} from './DarkModeBadgeButton';
+import * as React from "react";
+import { Badge } from "@twilio-paste/badge";
+import { Box } from "@twilio-paste/box";
+import {
+  Modal,
+  ModalBody,
+  ModalHeader,
+  ModalHeading,
+} from "@twilio-paste/modal";
+import { NewIcon } from "@twilio-paste/icons/esm/NewIcon";
+import { DarkModeFAQModal } from "./DarkModeFAQModal";
 
 export const DarkModeBadge: React.FC = () => {
+  // Modal properties
   const [isOpen, setIsOpen] = React.useState(false);
+
   return (
     <Box display="flex" alignItems="center" marginLeft="space40">
-      <DarkModeBadgeButton
-        onClick={() => {
-          setIsOpen(true);
-        }}
+      <Badge as="button" variant="new" onClick={() => setIsOpen(true)}>
+        New
+        <NewIcon size="sizeIcon10" decorative={false} title="Find out more" />
```

Comment:
Suuuuupuer nitpicky but should the icon go in front of the text? Does it matter?

Reply:
Oh, lol, yes, we just added a line about putting the icon in front of the text in Badge as part of the Stage 2 updates. Good catch!